### PR TITLE
Moved the mesh folder to avoid the having to type pixi.extras.mesh.Mesh, plus tried getter-setters

### PR DESCRIPTION
**Pull request by alvinsight**

### CODE REVIEW

File: src/deprecation.js [hunk 1]
```diff
@@ -17,15 +13,33 @@ core.AssetLoader = function () {
...
+
+    Strip: {
+        get: function ()
+        {
+            window.console.warn('The Strip class has been renamed to Mesh, please use Mesh from now on');
+            return new mesh.Mesh(texture);
+        }
+    }
+
+});
```
Comment (englercj) [inline src/deprecation.js]: Is there a reason we want these to be non-writable?

**Comment by englercj** [comment]
:+1:

**Comment by GoodBoyDigital** [comment]
nice one Alvin! :+1:

**Comment by englercj** [comment]
Why did we change some deprecated things to be non-writable properties and leave the other ones as-is?

**Comment by GoodBoyDigital** [comment]
Yo! Its not to make them private. The purpose is so that anything that extends a DisplayObjectContainer would break. The below code now works ( but still outputs the deprecation message )

```
var myThing = function()
{
    PIXI.DisplayObjectContainer.call(this);
}

myThing.prototype = Object.create( PIXI.DisplayObjectContainer );
```

**Comment by englercj** [comment]
@GoodBoyDigital Huh? That doesn't make sense, defining it as a property or adding it as a property shouldn't be any different in this case (also object.create should use the prototype). I find it hard to believe that this change fixes anything.

**Comment by GoodBoyDigital** [comment]
Good catch @englercj . @alvinsight are you cool to tweak? We need to remove the 'new' keywords from the getters. Cheers dude!

**Pull request merged**